primitives/src/lib.rs

```diff
@@ -101,7 +101,7 @@ pub const DEFAULT_FINALITY_VERSION: Version = 0;
 /// Current version of abft.
 pub const CURRENT_FINALITY_VERSION: u16 = LEGACY_FINALITY_VERSION + 1;
 /// Legacy version of abft.
-pub const LEGACY_FINALITY_VERSION: u16 = 1;
```

Comment:
Does it affect the release procedure anyhow? Ilke we need to issue manualy some ext?

Reply:
I'd say that running update-nets from release branches is one thing to do, and I believe that it would be meaningful to run a kind of update-net in which we start from `main` and only a **part** of nodes (say half) is updated.
Maybe @kostekIV or @timorleph would have some other suggestions.

Explanation: version to use is stored on the chain and currently it is `2` which is translated to using `CURRENT` version. I altered the codebase so that the `LEGACY` version corresponds now to what was the `CURRENT` version. So changing `LEGACY` from `1` to `2` **shouldn't cause any difference in node's behavior**.

Reply:
To answer the question directly – yes, we will need to set the finality version on-chain as probably the final part of the update, after we make sure ~all external nodes are updated as well.

For testing, I thought there were already some tests checking this type of update, but I might be wrong; if there are none, there should be a task for creating them. For manual testing we will have to run this somehow as well.

Reply:
Ok, noted. Thanks.